### ISSUE TITLE
FOOTER : A11Y violations "color contrast" 

### ISSUE THREAD

**Issue by iRohitSingh** [kitconcept/volto-light-theme#309]
as admin/user

go to https://light-theme.kitconcept.io/
 
 Run axe DevTool
 
 
 
<img width="1364" alt="Screenshot 2023-12-19 at 4 40 04 PM" src="https://github.com/kitconcept/volto-light-theme/assets/61353484/6f7f3c33-8245-49de-89ed-51372ebb7f74">

currently we are using  #007eb1 color. (Blue color)

**Comment by iRohitSingh** [comment]
Fix: pr: https://github.com/kitconcept/volto-light-theme/pull/300/commits/0ebd3f9c8aed51c655ed3761018d0a1b60e194e3

**Comment by tisto** [comment]
@iRohitSingh do we have automated Cypress/Axe tests that show this violation? Please first create a PR with a test that fails and then fix the issue.

**Comment by ankiiisharma** [comment]
hey @tisto , is there any "good first issue" to start my open-source journey?

**Comment by iRohitSingh** [comment]
> @iRohitSingh do we have automated Cypress/Axe tests that show this violation? Please first create a PR with a test that fails and then fix the issue.

@tisto We currently don't have infrastructure for adding th a11y test. Currently I am working on this branch which supports a11y. We already have a cypress test in which footer will fail see this link https://github.com/kitconcept/volto-light-theme/pull/300/files#diff-3667b73583aa42400bbbcaeef7073c08017d0ff28e54797ef74a112c70c7a344R10

Once this pr is merged we will have a cypress test for footer as well as fix.

**Comment by iRohitSingh** [comment]
@ankiiisharma hey, Please setup the project and play with this addons. If you think something is looking odd or broken please create a issue and start working on.

**Comment by ankiiisharma** [comment]
@iRohitSingh how can i connect with you?

**Comment by iRohitSingh** [comment]
Fixed.
